### ISSUE TITLE
C4AI Gradient: Work with dev to determine how we can tokenize the AI gradient requirements

### ISSUE THREAD

**Issue by mattrosno** [carbon-design-system/carbon#15073]
**Blocked by**

- #15072 
```[tasklist]
### Tasks
- [ ] Sync between design and dev on token naming
```

**Comment by mattrosno** [comment]
@aagonzales @tw15egan I'm thinking we should keep this issue open until we revisit token naming:

https://github.com/carbon-design-system/carbon/pull/15081#discussion_r1386930288

**Comment by mattrosno** [comment]
@tw15egan @aagonzales now that the AI gradient has been updated and rolled out across our AI components - how do we feel about this issue? Is there more work to be done, or can we close this issue (and the gradient epic)?

**Comment by tw15egan** [comment]
I haven't removed any tokens that are no longer needed after the Phase 2 updates. Once I get `Modal` updated, I can clean up the tokens and remove unused ones since that is the last

**Comment by mattrosno** [comment]
@tw15egan @aagonzales I'm going to close this as not planned. Once we know the next steps for the gradient if there are more changes, we should detail out a new issue or two for that.
